### PR TITLE
[Task-23]  Create eCommerce menu

### PR DESCRIPTION
**Pull request by rajiravenantech**
Create eCommerce menu

### CODE REVIEW

File: src/app/components/Ecommerce/AddNewProduct.tsx [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+            const reader = new FileReader();
+
+            reader.onloadend = () => {
+                setFormData((prevFormData) => ({
+                    ...prevFormData,
+                    [name]: reader.result,
+                    imageFileName: file ? file.name : "",
+                }));
+            };
+            console.log('file',reader.result)
```
Comment (shanmugapriyaravenantech) [inline src/app/components/Ecommerce/AddNewProduct.tsx:105]: Remove console.log

File: src/app/components/Ecommerce/AddNewProduct.tsx [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+        if (formData.name.trim() === "") {
+            newErrors.name = "Name is required";
+            isValid = false;
+        } else {
+            newErrors.name = "";
+        }
+
+        if (formData.compareAtPrice.trim() === "") {
+            newErrors.compareAtPrice = "compare at price is required";
```
Comment (shanmugapriyaravenantech) [inline src/app/components/Ecommerce/AddNewProduct.tsx:60]: Make 'compare' first letter to be in capital for error message

File: src/app/components/Ecommerce/DeleteModal.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Modal, Button, Box } from "@mui/material";
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+const DeleteModal = ({ open, onClose, onDeleteConfirm, rowData,orderId }) => {
```
Comment (shanmugapriyaravenantech) [inline src/app/components/Ecommerce/DeleteModal.tsx:6]: rowData can be removed as it is not been used

File: src/app/page.module.css [hunk 4]
```diff
@@ -229,3 +229,13 @@
     transform: rotate(0deg);
   }
 }
+
+@media screen and (max-width: 500px) {
```
Comment (shanmugapriyaravenantech) [inline src/app/page.module.css:233]: Remove unwanted styles

File: src/app/components/Ecommerce/AddNewProduct.tsx (hunk 1 above)
Comment (rajiravenantech) [reply src/app/components/Ecommerce/AddNewProduct.tsx]: Removed console.log

File: src/app/components/Ecommerce/DeleteModal.tsx (hunk 3 above)
Comment (rajiravenantech) [reply src/app/components/Ecommerce/DeleteModal.tsx]: Removed `rowData`  in deletemodal

File: src/app/page.module.css (hunk 4 above)
Comment (rajiravenantech) [reply src/app/page.module.css]: Removed unwanted styles

File: src/app/components/Ecommerce/AddNewProduct.tsx (hunk 2 above)
Comment (rajiravenantech) [reply src/app/components/Ecommerce/AddNewProduct.tsx]: Changed error message in `compareAtPrice`

**Review by shanmugapriyaravenantech — APPROVED** [review]

**Review by esakkirajaravenantech — APPROVED** [review]

**Pull request merged**